test/core/gprpp/status_helper_test.cc

```diff
@@ -69,7 +69,7 @@ TEST(StatusUtilTest, GetStrNotExistent) {
 
 TEST(StatusUtilTest, SetAndGetTime) {
   absl::Status s = absl::CancelledError();
-  absl::Time t = absl::Now();
+  absl::Time t = absl::FromUniversal(absl::ToUniversal(absl::Now()));
```

Comment:
can you add some comments to explain this please?

Reply:
I changed this. PTAL.